core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -761,7 +756,7 @@ func TestLogPoller_PollAndSaveLogs(t *testing.T) {
 			th := SetupTH(t, tt.finalityTag, tt.finalityDepth, 3, 2, 1000)
 
 			// Set up a log poller listening for log emitter logs.
-			err := th.LogPoller.RegisterFilter(logpoller.Filter{
+			err := th.LogPoller.RegisterFilter(testutils.Context(t), logpoller.Filter{
```

Comment:
Like in one of the earlier tests, think it'd be better to defined a `ctx := testutils.Context(t)` at the top and use that everywhere below